charts/rancher-gke-operator/1.0.100/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.7
```

Comment:
Hmm I thought this was right before. Should be 2.5.7-alpha1. Ex: https://github.com/rancher/system-charts/blob/dev-v2.5/charts/rancher-eks-operator/1.0.100/questions.yaml

Reply:
Fixed